packages/xo-server/src/xo-mixins/proxies.js

```diff
@@ -202,7 +208,10 @@ export default class Proxy {
 
       await xapi.startVm(vm.$id)
     } catch (error) {
-      await xapi._deleteVm(vm)
+      await Promise.all([
+        app.unbindLicense({ licenseId, boundObjectId: vm.uuid }),
```

Comment:
What if the license was not successfully bound?

Reply:
I'll use [`waitAll`](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-server/src/_waitAll.js) to wait the two promises.

Reply:
My question was not related to `Promise.all`.

Reply:
It'll throw an error. I don't handled this case because the operation will fails anyway but if we want to get a proper error, i need to test if the error is about the license binding.